### PR TITLE
Formatting

### PR DESCRIPTION
**Pull request by thehappydinoa**
- PEP8 Format
- Organized imports
- Added link to `README.md`
- Formatted html and css

### CODE REVIEW

File: tracking.py [hunk 1]
```diff
@@ -130,25 +143,25 @@ def receive_buffer(tweets):
...
-               |    |  \ \/ \/ /  |  \   __\/ __ \_  __ \                          
-               |    |   \     /|  |  ||  | \  ___/|  | \/                          
-               |____|    \/\_/ |__|__||__|  \___  >__|                             
-            .___        __         .__  .__  .__\/                                 
-            |   | _____/  |_  ____ |  | |  | |__| ____   ____   ____   ____  ____  
-            |   |/    \   __\/ __ \|  | |  | |  |/ ___\_/ __ \ /    \_/ ___\/ __ \ 
-            |   |   |  \  | \  ___/|  |_|  |_|  / /_/  >  ___/|   |  \  \__\  ___/ 
+    print(colored('''\n\t\t\033[1m
+             ___________       .__.__  __
+             \__    ___/_  _  _|__|__|/  |_  ___________
```
Comment (batuhaniskr) [inline tracking.py]: Tool Logo is incorrected format when running tracking.py. You can look https://ibb.co/cN0hfV link.

Comment (thehappydinoa) [reply tracking.py]: Ah let me check that out.

Comment (thehappydinoa) [reply tracking.py]: Fixed

File: analysis.py [hunk 2]
```diff
@@ -62,7 +66,8 @@ def analysis_user():
     with sqlite3.connect(db_path) as db:
         conn = db
         c = conn.cursor()
-        c.execute("SELECT  username, count(*) as tekrar FROM Tweet  group by username order by tekrar desc LIMIT 10")
+        c.execute(
+            "SELECT  username, count(*) as tekrar FROM Tweet  group by username order by tekrar desc LIMIT 10")
```
Comment (batuhaniskr) [inline analysis.py]: Why was it add blank line in c.execute?

File: analysis.py [hunk 3]
```diff
@@ -53,7 +56,8 @@ def main(argv):
 def map():
     location = location_analysis()
     api_key = settings.GOOGLE_MAP_API_KEY
-    url = 'https://maps.googleapis.com/maps/api/js?key=' + api_key + '&libraries=visualization&callback=initMap'
+    url = 'https://maps.googleapis.com/maps/api/js?key=' + \
+        api_key + '&libraries=visualization&callback=initMap'
```
Comment (batuhaniskr) [inline analysis.py]: There is also a blank line here.

File: gui/socialgui.py [hunk 4]
```diff
@@ -64,15 +62,18 @@ def setupUi(self, So):
         font.setBold(True)
         font.setWeight(75)
         self.pushButton_2.setFont(font)
-        self.pushButton_2.setStyleSheet(_fromUtf8("background-color:rgb(65, 65, 65); color:rgb(255, 255, 255);"))
+        self.pushButton_2.setStyleSheet(
```
Comment (batuhaniskr) [inline gui/socialgui.py]: +

**Comment by thehappydinoa** [comment]
This is just PEP8 formatting.

**Pull request closed without merging**